packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,210 +1,202 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-border-color: Specify the border color of the component
+ * @prop --calcite-accordion-text-color: Specify the text color of the component
+ * @prop --calcite-accordion-text-color-highlighted: Specify the highlighted text color
```

Comment:
What is this `highlighted` property for?

Reply:
token naming consistency across components. "highlighted" is a name for elements we want to draw a user's attention to but may not be "selected".